logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/FeatureConfigEventReceiver.kt

```diff
@@ -81,7 +81,33 @@ internal class FeatureConfigEventReceiverImpl internal constructor(
                 userConfigRepository.setConferenceCallingEnabled(conferenceCallingEnabled)
             }
 
+            is Event.FeatureConfig.GuestRoomLinkUpdated -> {
+                handleGuestRoomLinkFeatureConfig(event.model.status)
+            }
+
             is Event.FeatureConfig.UnknownFeatureUpdated -> kaliumLogger.w("Ignoring unknown feature config update")
         }
     }
+
+    private fun handleGuestRoomLinkFeatureConfig(status: Status) {
+        if (!kaliumConfigs.guestRoomLink) {
+            userConfigRepository.setGuestRoomStatus(false, null)
```

Comment:
Could you explain a bit more to me here why we are setting it to false if we don't have `kaliumConfigs.guestRoomLink`?

Reply:
`kaliumConfigs` will return the default value from `default.json`, if it's false then we disable the feature